presto-common/src/main/java/com/facebook/presto/common/predicate/TupleDomainFilter.java

```diff
@@ -187,11 +192,25 @@ public boolean isPositionalFilter()
         {
             return false;
         }
+
+        @Override
+        public long getRetainedSizeInBytes()
+        {
+            return 0;
+        }
     }
 
     class AlwaysFalse
             extends AbstractTupleDomainFilter
     {
+        private static final int INSTANCE_SIZE = ClassLayout.parseClass(AlwaysFalse.class).instanceSize();
+
+        @Override
+        public long getRetainedSizeInBytes()
```

Comment:
Can you please move this method in each filter class to the last of the public with @Override method? The definition of the methods should match the order they are defined in the interface.